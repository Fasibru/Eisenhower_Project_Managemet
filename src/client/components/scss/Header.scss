@import './variables';

.header {
  grid-area: header;
  background-color: $component-color-HeaderFooter;
  color: $element-color;
  position: sticky;
  top: 0;
}