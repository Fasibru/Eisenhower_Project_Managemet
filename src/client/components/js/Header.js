import React from 'react';

import '../scss/Header.scss';

import User from './User';

const Header = () => (
  <header className="header">
    <p>Statistics will follow here</p>
    <User />
  </header>
);

export default Header;
