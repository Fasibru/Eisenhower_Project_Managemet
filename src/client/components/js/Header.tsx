import * as React from 'react';

import '../scss/Header.scss';

import User from './User';

// tslint:disable-next-line: variable-name
const Header = () => (
  <header className="header">
    <p className="header__statistics">Statistics will follow here</p>
    <User />
  </header>
);

export default Header;
