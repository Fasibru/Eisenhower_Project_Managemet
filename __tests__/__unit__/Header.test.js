import React from 'react';
import renderer from 'react-test-renderer';
import Header from '../../src/client/components/js/Header';

describe('Testing Header', () => {
  it('Renders correctly', () => {
    const header = renderer
      .create(<Header />);

    expect(header).toMatchSnapshot();
  });
});
