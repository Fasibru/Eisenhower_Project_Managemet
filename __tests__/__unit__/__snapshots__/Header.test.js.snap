// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Header Renders correctly 1`] = `
<header
  className="header"
>
  <p>
    Show Statistics
  </p>
  <p>
    Show Statistics
  </p>
</header>
`;
